backend/scripts/seed_demo_data.py

```
@@ -0,0 +1,494 @@
+        line_end = line_start + random.randint(0, 15)
+
+        crit_weight = tgt.criticality_weight
+        sev_weight = {Severity.CRITICAL: 5, Severity.HIGH: 4, Severity.MEDIUM: 3, Severity.LOW: 2, Severity.INFO: 1}[sev]
+        epss_multiplier = (epss or 0.1) * 10
+        kev_bonus = 20 if kev else 0
+        priority = min(99, int(crit_weight * sev_weight * 3 + epss_multiplier + kev_bonus))
+
+        days_first_seen = random.randint(1, 80)
+        f_seen = now - timedelta(days=days_first_seen, hours=random.randint(1, 20))
+        l_seen = now - timedelta(hours=random.randint(1, 24))
+        mit_at = now - timedelta(days=random.randint(1, 10)) if state == FindingState.MITIGATED else None
```

Ruff (0.16.3)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)